specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/storage.json

```diff
@@ -3752,6 +3770,18 @@
           "$ref": "#/definitions/StorageAccountSkuConversionStatus",
           "x-ms-client-name": "StorageAccountSkuConversionStatus",
           "description": "This property is readOnly and is set by server during asynchronous storage account sku conversion operations."
+        },
+        "dnsEndpointType": {
+          "type": "string",
+          "enum": [
+            "Standard",
+            "AzureDnsZoned"
+          ],
+          "x-ms-enum": {
+            "name": "DnsEndpointType",
+            "modelAsString": true
+          },
+          "description": "Specifies the type of DNS Endpoint with which the Storage Account should be accessed."
```

Comment:
Please give more explanation for the impact of the 2 options. 
The current description looks not enough to make user understand  the function of the property.

Reply:
I suggest - "Allows you to specify the type of endpoint. Set this to AzureDNSZone to create a large number of accounts in a single subscription. "

Reply:
done